pages/common/django-admin.md

```diff
@@ -0,0 +1,20 @@
+# django-admin
+
+> Django’s utility for administrative tasks.
+> More information: <https://docs.djangoproject.com/en/3.0/ref/django-admin/>.
+
+- Create a new django project:
+
+`django-admin startproject {{project_name}}`
+
+- Create a new app for the current project:
+
+`django-admin startapp {{app_name}}`
+
+- Check the current version of Django:
+
+`django-admin --version`
+
+- Check a specific help page of Django:
```

Comment:
```suggestion
- Display more information of the given command:
```

Reply:
```suggestion
- Display more information for the given command:
```
how about this?

Reply:
Great. I'll go along with @zdroid's suggestion. Thanks @andrik, this is much better.